Apps/APN/ios/AppDelegate.h

```diff
@@ -0,0 +1,10 @@
+#import <React/RCTBridgeDelegate.h>
+#import <UIKit/UIKit.h>
+#import <UserNotifications/UNUserNotificationCenter.h>
+#import <UserNotifications/UserNotifications.h>
+
+@interface AppDelegate : UIResponder <UIApplicationDelegate, RCTBridgeDelegate, UNUserNotificationCenterDelegate>
```

Comment:
Is it common for RN apps to use objective-c AppDelegate files?

Reply:
Yes, react native apps commonly use ObjC code base.